proto/minder/v1/minder.proto

```diff
@@ -2072,11 +2164,23 @@ message RuleType {
         message Eval {
             // type is the type of the data evaluation.
             // Right now only `jq` is supported as a driver
-            string type = 1;
+            string type = 1 [
+                (buf.validate.field).string = {
+                    pattern: "^[a-z]+(_[a-z]+)*$",
+                    min_len: 1,
+                    max_len: 200,
+                }
+            ];
 
             message JQComparison {
                 message Operator {
-                    string def = 1;
+                    string def = 1 [
+                        (buf.validate.field).string = {
+                            pattern: "^\\.[a-z_]+(\\.[a-z_]+)*$",
```

Comment:
Hey, this would have fixed my bad ruletype debugging experience!  Thank you!

Reply:
Do we need to accommodate `"` or array access (`[]`)?

Reply:
Fixed 👍